hedera-node/src/main/java/com/hedera/services/queries/crypto/GetAccountRecordsAnswer.java

```diff
@@ -112,6 +153,6 @@ public HederaFunctionality canonicalFunction() {
 	@Override
 	public Optional<SignedTxnAccessor> extractPaymentFrom(final Query query) {
 		final var paymentTxn = query.getCryptoGetAccountRecords().getHeader().getPayment();
-		return Optional.ofNullable(SignedTxnAccessor.uncheckedFrom(paymentTxn));
+		return Optional.of(SignedTxnAccessor.uncheckedFrom(paymentTxn));
```

Comment:
What was the significance of this change?

Reply:
The gRPC accessors never return `null`, so IntelliJ was marking this `Optional.ofNullable()` as a code smell.